### PR TITLE
add config for the CPCOLLAB project

### PR DESCRIPTION
**Pull request by mikebonnet**

### CODE REVIEW

File: Makefile.cpcollab [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	venv/bin/pip install -Ur requirements.txt
+
+buildstatus: install
+	venv/bin/python finishline \
+		--scenario statusslides \
+		--config cpcollab.conf > foo.md
+
+uploadstatus: buildstatus
+	./node_modules/md2gslides/bin/md2gslides.js foo.md \
+		-t "C3I Status" \
```
Comment (accorvin) [inline Makefile.cpcollab]: You probably want to change this to something different. Also does the -a value for the slides ID on the next line map to a new status deck, or the preexisting C3I one?

**Comment by accorvin** [comment]
@mikebonnet out of curiosity, did you review any of the getting started notes in this project's README? If so, were they helpful? Any ideas that aren't clear or need more info?

File: Makefile.cpcollab (hunk 1 above)
Comment (mikebonnet) [reply Makefile.cpcollab]: Good catch, updated. Yep, the -a references a new slide deck: https://docs.google.com/presentation/d/15bXAWuHOQiy4nTfTvztfVgXrO6vHL9XuVuP_4YKjEBo/edit

**Comment by mikebonnet** [comment]
@accorvin Yes, I reviewed the README. It was helpful, I don't have any suggestions for improvements at the moment.

**Pull request merged**